testing-modules/testng_command_line/src/test/java/com/baeldung/testing_modules/testng_command_line/DateSerializerServiceUnitTest.java

```diff
@@ -0,0 +1,50 @@
+package com.baeldung.testing_modules.testng_command_line;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+import org.testng.Assert;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+@Test(testName = "Date Serializer")
+public class DateSerializerServiceUnitTest {
+
+    private DateSerializerService toTest;
+
+    public DateSerializerServiceUnitTest() {
+        super();
+    }
```

Comment:
Remove this redundant constructor. It's the default constructor, and we don't need it or the `super()`